src/Relation/Binary/Construct/SetoidFromPartialSetoid.agda

```diff
@@ -0,0 +1,61 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Conversion of a PartialSetoid into a Setoid
+------------------------------------------------------------------------
+
+open import Relation.Binary.Bundles using (PartialSetoid; Setoid)
+
+module Relation.Binary.Construct.SetoidFromPartialSetoid
+  {a ℓ} (S : PartialSetoid a ℓ) where
+
+open import Data.Product.Base using (_,_; Σ; proj₁; proj₂)
```

Comment:
I don't know if it would be worse, or better, to have something like
```suggestion
open import Data.Product.Base using (_,_; Σ)
  renaming (proj₁ to ι; proj₂ to x≈x)
```
so that subsequent definitions can exploit such 'abstract' vocabulary/notation, rather than expose the concrete projection names?

Reply:
In the interests of clarity: are you saying that you would prefer that it be changed in favour of the `record` version?

Reply:
Yes